test/core/IMangroveAbi.t.sol

```diff
@@ -3,23 +3,30 @@ pragma solidity ^0.8.10;
 
 import {Test2} from "mgv_lib/Test2.sol";
 import {stdJson} from "forge-std/StdJson.sol";
+import "mgv_lib/Debug.sol";
+import {Mangrove} from "mgv_src/Mangrove.sol";
+import {MgvAppendix} from "mgv_src/MgvAppendix.sol";
 
 using stdJson for string;
 
+abstract contract MangroveMix is Mangrove, MgvAppendix {
+// constructor() Mangrove(address(0),0,0) {}
+}
+
 contract IMangroveAbiTest is Test2 {
   function test_abi_is_identical() public {
     string memory root = vm.projectRoot();
-    string memory mangrovePath = string.concat(root, "/out/Mangrove.sol/Mangrove.json");
+    string memory mangrovePath = string.concat(root, "/out/IMangroveAbi.t.sol/MangroveMix.json");
     string memory interfacePath = string.concat(root, "/out/IMangrove.sol/IMangrove.json");
     string memory mangroveJson = vm.readFile(mangrovePath);
     string memory interfaceJson = vm.readFile(interfacePath);
 
-    string[] memory mangroveAbiElementsWithConstructor = stdJson.readStringArray(mangroveJson, ".abi");
-    string[] memory mangroveAbiElements = new string[](mangroveAbiElementsWithConstructor.length-1);
+    string[] memory mangroveAbiElements = stdJson.readStringArray(mangroveJson, ".abi");
+    // string[] memory mangroveAbiElements = new string[](mangroveAbiElementsWithConstructor.length-1);
     // Delete constructor definition as it is not present in interface
-    for (uint i; i < mangroveAbiElements.length; ++i) {
-      mangroveAbiElements[i] = mangroveAbiElementsWithConstructor[i + 1];
-    }
+    // for (uint i; i < mangroveAbiElements.length; ++i) {
+    //   mangroveAbiElements[i] = mangroveAbiElementsWithConstructor[i + 1];
+    // }
```

Comment:
Delete?

Reply:
Hopefully not as that would disable the test :D

Reply:
huh? Isn't it just the code that removes constructor that has been commented out?

Reply:
yes!